dms-service/README.md

```
@@ -0,0 +1,56 @@
+
+### DMS
+
+Further documentation regarding the DMS can be found here (internal only):
+- https://confluence.muenchen.de/display/KM53/REST-EAI-Schnittstelle
+- https://dmsresteai-dev-dmsresteai.apps.capk.muenchen.de/swagger-ui/index.html
+
+## Development
+
+- The dms-service is built with JDK21
+- For local development and testing the dev docker-compose stack of the [dispatch-service](../dispatch-service) can be used.
+    - Can be started with `docker compose up -d`
```

Markdownlint (0.37.0)

Expected: 2; Actual: 4
Unordered list indentation

(MD007, ul-indent)